backend/qemu.pm

```diff
@@ -734,28 +747,18 @@ sub start_qemu ($self) {
         # of ports is 32 so enough for 14 workers per host.
         $vars->{VDE_PORT} ||= ($vars->{WORKER_ID} // 0) * 2 + 2;
     }
-
-    # arch discovery
-    my $arch = $vars->{ARCH} // '';
```

Comment:
Storing the arch just once in a local variable made actually more sense than the repeated hash accesses and the repeated `// ''` operations. (But I don't really care.)

Reply:
sure. I re-introduced the arch variable